device-types/DataDirect Networks/SFA200NVX2-Controller.yaml

```diff
@@ -0,0 +1,30 @@
+---
+manufacturer: DataDirect Networks
+model: SFA200NVX2 Controller
+slug: datadirect-networks-sfa200nvx2-controller
+comments: Controller for SFA200NVX2
+part_number: CTLR-200NVX2E
+u_height: 0
+is_full_depth: false
+airflow: front-to-rear
```

Comment:
Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

Reply:
The weight of the controller alone has not been announced.